packages/analytics/src/types.ts

```diff
@@ -22,3 +22,8 @@ export type Event = {
         [key: string]: string | string[] | number | number[] | boolean | null;
     };
 };
+
+export type ReportConfig = {
+    anonymize?: boolean; // Log event without session and device ID to prevent linking it to other events.
```

Comment:
Device Id was never tracked 🤯

`instanceId` and `sessionId` are attributes you are talking about